plot.py

```diff
@@ -375,16 +379,18 @@ def plot(self, df_in, output_prefix, output_dir, dpi, force, show, verbose):
 
         # Decide if using internal legend.
         external_legend = self.legend_outside
-
+        filled_markers = ('o', 'v', '^', '<', '>', '8', 's', 'p', '*', 'h', 'H', 'D', 'd', 'P', 'X')
         g = None
         if self.plot_type == "lineplot":
+            # plot the data @todo - lineplot vs scatter?
             g = sns.lineplot(
                 data=df, 
                 x=self.xkey, 
                 y=self.ykey, 
                 hue=self.huekey, 
                 style=self.stylekey, 
-                markers=True,
+                markers=False,
+                dashes = False,
```

Comment:
With markers and dashes disabled series are no longer distinguishable (colour alone is insufficient for visually impaired viewers / black and white)

Reply:
fixed